buildlib/pr/io_demo/io-demo.yml

```diff
@@ -1,7 +1,7 @@
 parameters:
   - name: demands
     type: string
-    default: "ucx_iodemo -equals yes"
+    default: "ucx_iodemo_rain -equals yes"
```

Comment:
maybe "ucx_roce_iodemo" label?

Reply:
no no, it's for test on rain, will revert back to 'ucx_iodemo'